galasa-parent/dev.galasa/pom.xml

```diff
@@ -11,6 +11,15 @@
 	<artifactId>dev.galasa</artifactId>
 	<version>0.13.0-SNAPSHOT</version>
 	<packaging>bundle</packaging>
+	
+	<dependencies>
```

Comment:
NO WAY.
You have created a dependency loop here.   dev.galasa.framework is dependent on dev.galasa,   so dev.galasa cannot be dependent on dev.galasa.framework